packages/transformers/js/core/src/global_replacer.rs

```diff
@@ -70,6 +70,7 @@ impl<'a> Fold for GlobalReplacer<'a> {
               attributes: None,
               is_optional: false,
               is_helper: false,
+              source_type: Some(SourceType::Module),
```

Comment:
Should this be `None` as in `fs.rs`?

Reply:
file dependencies don't need a source type, but in this case it's referring to a require that should be treated as a module.